Source/SVGKImage.m

```diff
@@ -183,7 +192,7 @@ +(SVGKParser *) imageWithSource:(SVGKSource *)source onCompletion:(SVGKImageAsyn
 	 */
 	
 	SVGKParser* parser = [SVGKParser newParserWithDefaultSVGKParserExtensions:source];
-	dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0),
+	dispatch_async(serialQueue,
```

Comment:
We use here this static `serialQueue` and it's only created here: https://github.com/thefabulous/SVGKit/pull/1/files#diff-8516b189f592358e513aa3b513bfc90eR174 which is inside the pragma block:

```
#if ENABLE_GLOBAL_IMAGE_CACHE_FOR_SVGKIMAGE_IMAGE_NAMED
[...]
#endif
```

so if `ENABLE_GLOBAL_IMAGE_CACHE_FOR_SVGKIMAGE_IMAGE_NAMED` is disabled we would have `nil` queue and completion won't be executed.

Reply:
Good catch, thanks!